features/frontend/products.feature

```diff
@@ -9,18 +9,21 @@ Feature: Products
           And there are following taxonomies defined:
             | name     |
             | Category |
-          And taxonomy "Category" has following taxons:
+          And there are following users:
+            | email       | password | enabled |
+            | bar@foo.com | foo1     | yes     |
+        And taxonomy "Category" has following taxons:
             | Clothing > T-Shirts     |
             | Clothing > PHP T-Shirts |
             | Clothing > Gloves       |
-          And the following products exist:
-            | name             | price | taxons       |
-            | Super T-Shirt    | 19.99 | T-Shirts     |
-            | Black T-Shirt    | 18.99 | T-Shirts     |
-            | Sylius Tee       | 12.99 | PHP T-Shirts |
-            | Symfony T-Shirt  | 15.00 | PHP T-Shirts |
-            | Doctrine T-Shirt | 15.00 | PHP T-Shirts |
-          And there are following channels configured:
+        And the following products exist:
+            | name             | price | taxons       | average rating |
```

Comment:
Unfortunately, calculating average rating is connected with state machine callback, so we should create reviews and change state on them properly, to trigger this... It's a little bit exaggeration for simple test, I suppose.